bin/linters.sh

```diff
@@ -92,6 +92,6 @@ run_yaml_lint
 check_grafana_dashboards
 check_samples
 
-
 "${WORKSPACE}/scripts/run_golangci.sh"
 "${WORKSPACE}/scripts/check_license.sh"
+"${WORKSPACE}/scripts/check_dockerfiles.sh"
```

Comment:
linters changes need to go in pkg.  I'll take a look at the rest of this PR and if its sufficent I'll approve, but please do move the linter code to pkg if not already there.

Reply:
@geeknoid told me last week that it wouldn't be ran on anything in pkg/ , ( see istio/pkg#35 ), would you mind clarifying this together?

Reply:
* Common scripts should go to istio/common-files.

* Common tool source code should go to istio/tools.

* Common tool binaries should be produced in istio/tools and published as containers.

* Other repos either use the script that gets copied from istio/common-files or the containers produced in istio/tools.

That's the plan of course. We have quite  bit of shuffling to do to get to  that state.

Reply:
@geeknoid 2 questions then:
1. What about pkg? (what should go to it?)
2. Does that mean the check_dockerfile.sh script should land into istio/tools too ?

Reply:
istio/pkg is intended for utility packages used by a variety of components. This is intended for utility package with general purpose use. We want to keep the number of dependencies pulled in by this repo to a minimum since it sits at the bottom of everything. So this should not depend on k8s for example.

Per my first  point above, check_dockerfile.sh should be checked into istio/common-files. As we replicate common files to all repos, the script will eventually show up in istio/tools/scripts where it can be used to verify the docker files used to produce our tool cotainers.

Reply:
@fpesce I meant istio/common-tools - apologies for the counfusion.